Source/WebCore/platform/graphics/cv/VideoFrameCV.mm

```diff
@@ -503,18 +503,18 @@ static void copyPlane(uint8_t* destination, const uint8_t* source, uint64_t sour
     auto height = size.height();
 
     auto dataBaseAddress = pixelBuffer->bytes().data();
-    auto leakedBuffer = &pixelBuffer.leakRef();
+    RefPtr leakedBuffer = &pixelBuffer.leakRef();
```

Comment:
It's still not perfect but I think this would look a little less weird than assigning the `leakRef()` output to a RefPtr.
1. Drop this variable here.